Feature - import API Management policy to an operation script (#52)
arcus-azure/arcus.scripting: docs/preview/features/powershell/azure-api-management.md

PS> Create-AzApiManagementApiOperation -ResourceGroup $ResourceGroup -ServiceName $ServiceName -ApiId $ApiId -OperationId $OperationId -Method $Method -UrlTemplate $UrlTemplate -OperationName $OperationName -Description $Description -PolicyFilePath $PolicyFilePath
# New API operation '$OperationName' on API Management service was added.
```	

## Import policy to an operation in the API Management service
Imports a policy from a file to an API operation in Azure API Management.

| Parameter           | Mandatory | Description                                                                                              |
| ------------------- | --------- | -------------------------------------------------------------------------------------------------------- |
| `ResourceGroupName` | yes       | The resource group containing the API Management service                                                 |
| `ServiceName`       | yes       | The name of the API Management service located in Azure                                                  |
| `ApiId`             | yes       | The ID to identify the API running in API Management                                                     |

> | `OperationId`       | yes       | The ID to identify the to-be-imported policy operation on the API                                        |

Reviewer: ```suggestion
| `OperationId`       | yes       | The ID to identify the operation for which to import the policy                                        |
```

> | `PolicyFilePath`    | no        | The path to the file containing the optional policy of the to-be-imported policy operation on the API    |

Reviewer: ```suggestion
| `PolicyFilePath`    | yes        | The path to the file containing the to-be-imported policy     |
```
